Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -61,7 +61,7 @@ var SCROLLVIEW_REF = 'listviewscroll';
  *
  * ```
  * getInitialState: function() {
- *   var ds = new ListViewDataSource({rowHasChanged: (r1, r2) => r1 !== r2});
```

Comment:
This is going to get wiped by an internal sync but we have another diff that does the same thing so the change won't get lost.